deploy/crds/kieapp.crd.yaml

```diff
@@ -1871,6 +1871,41 @@ spec:
                                 to generated options in JAVA_OPTS. e.g. '-Dsome.property=foo'
                               type: string
                           type: object
+                        kafkaExt:
+                          description: KafkaExt kafka configuration to be used by
```

Comment:
IMO It would be better to use there Kafka integration naming then extension, as this configuration should server for Kie Server and Kafka integration.

Reply:
They used this naming